assets/src/components/detours/diversionPage.tsx

```diff
@@ -258,6 +260,25 @@ const DiversionPagePanelFooter = ({
   </div>
 )
 
+// If we just use the `dismissible` prop, the close button is
+// positioned absolutely in a way that looks weird, so we need to wrap
+// the Alert in our own show state logic.
+const RoutingErrorAlert = (): React.ReactElement => {
+  const [show, setShow] = useState<boolean>(true)
+
+  return (
+    <Alert
+      variant="ui-alert"
+      className="position-absolute top-0 left-0 mt-3 start-50 translate-middle-x icon-link z-1"
+      show={show}
+    >
+      <BsIcons.ExclamationTriangleFill />
+      Something went wrong. Please try again.
+      <CloseButton onClick={() => setShow(false)} />
+    </Alert>
+  )
+}
```

Comment:
suggestion/general-feedback(non-blocking): I'd prefer to try and make the `dismissible` prop work, but I'm open to making this feedback non-blocking and addressing it in a separate PR. I did some looking locally, and it seems there are two issues causing the close button to look weird

1. classes `border-box` and `inherit-box` are overriding bootstrap's preferred values.
	- Because `inherit-box` indiscriminately sets the box sizing value, when the close button requests a `box-sizing: content-box;` it's being ignored due to this. We'll likely solve this as apart of the CSS reboot changes. We could provide an override to `inherit-box` to make it stop applying to children of the alert.
2. Font size
	- When I was experimenting locally, I was still seeing some strange behaviour after I added `content-box` back. I think the issue here is that my browser's font size is set to 15, but somewhere on the page is overriding it to 16, but somehow both are coexisting and conflicting. This makes the close button overflow the alert container, looking a little strange. I'm pretty sure this only happens for me though because of my custom default user agent font size. This would also be fixed as apart of the reboot changes.
3. Another issue, I was finding that the close button's focus border was intentionally overflowing the alert, I believe for visual emphasis. The `contain: paint` value I set on the `__map` class seems to be the wrong approach, and should instead be converted to use the `isolation-isolate` utility class.

Reply:
I would also prefer just using `dismissible` if it's not too much effort to get it working! Right now I'm mainly focused on the interactions around automatically clearing the last waypoint on error, but I'll keep this in mind.